### PR TITLE
add global trigger warning

### PR DESCRIPTION
**Pull request by koilebeit**
# Pull request

## Proposed changes

First draft to solve #154 
To discuss:

- session storage or local storage
- design of the trigger warning

## Types of changes

- [x] New feature (non-breaking change which adds functionality).
- [ ] Enhancement (non-breaking change which enhances functionality)
- [ ] Bug Fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist

- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

## Summary by CodeRabbit

- **New Features**
	- Introduced a dynamic trigger warning modal for sensitive content, enhancing user awareness.
	- Updated trigger warning text to provide a more comprehensive understanding of historical materials.

- **Bug Fixes**
	- Removed outdated static alert for sensitive content from various pages, streamlining user experience.

- **Documentation**
	- Deleted the `sensible-inhalte.md` page, previously outlining sensitive content warnings and related topics.

### CODE REVIEW

File: _includes/js/trigger-warning-js.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<script>
+  // Function to hide the trigger warning using Bootstrap's modal method
+  function hideWarning() {
+    const modal = document.getElementById('triggerWarning');
+    const bootstrapModal = bootstrap.Modal.getInstance(modal) || new bootstrap.Modal(modal);
+    bootstrapModal.hide();
+
+    // Set in sessionStorage to not show the warning again for this session
```
Comment (maehr) [inline _includes/js/trigger-warning-js.html:8]: @koilebeit if possible change to LocalStorage

**Review by maehr — CHANGES_REQUESTED** [review]
@koilebeit Thanks for the PR! Nice work.

File: _includes/feature/trigger-warning.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  aria-labelledby="triggerWarningTitle"
+  aria-hidden="true"
+>
+  <div class="modal-dialog modal-fullscreen" role="document">
+    <div class="modal-content" style="display: flex; justify-content: center; align-items: center; height: 100vh;">
+      <div class="modal-body" style="max-width: 600px; text-align: center; margin: 0 auto;">
+        <h5 class="modal-title text-center w-100" id="triggerWarningTitle">Trigger Warnung</h5>
+        <p>{{ site.data.theme['trigger-warning-text'] | default: 'This collection may contain sensitive content.' }}</p>
+        <button id="proceedButton" type="button" class="btn btn-primary mt-4" aria-label="Hinweis ausblenden">
+          Verstanden
```
Comment (maehr) [inline _includes/feature/trigger-warning.html:15]: IMHO "Hinweis ausblenden" would be more clear than "Verstanden"

**Comment by mtwente** [comment]
I know the modal was supposed to cover the whole screen, but what do you think about the way this looks:

trigger_modal_85pct

I think it works well with the darkened background.

I added 85% opacity to the modal's backdrop. This way the users still get shown a glimpse of the website that they're on – or to put it differently: That they are in fact on a website that consists of more content than the trigger warning... And we have the logo + headline "Forschungsdatenplattform" already kind of visible in the background. The modal is centered horizontally and vertically on the screen.

**Comment by maehr** [comment]
> I know the modal was supposed to cover the whole screen, but what do you think about the way this looks:
> 
> trigger_modal_85pct
> I think it works well with the darkened background.
> 
> I added 85% opacity to the modal's backdrop, I could also go up to 90%. This way the users still get shown a glimpse of the website that they're on – or to put it differently: That they are in fact on a website that consists of more content than the trigger warning... And we have the logo + headline "Forschungsdatenplattform" already kind of visible in the background. The modal is centered horizontally and vertically on the screen.
> 
> P.S.: "Materialien _auf_ dieser Datenbank"? Not _in_ dieser Datenbank?

I like it 🥰 And. Also the the fixed typo

**Comment by mtwente** [comment]
What do you think about using the term *Content Note* instead of Triggerwarnung? It's less of a buzzword and more neutral/descriptive (see [here](https://beauftragte-missbrauch.de/fileadmin/user_upload/Content_Notes/Mit_Content_Notes_auf_sensible_Themen_aufmerksam_machen.pdf) or [here](https://www.cctl.cam.ac.uk/content-notes/what-are-content-notes).)

**Comment by maehr** [comment]
> What do you think about using the term _Content Note_ instead of Triggerwarnung? It's less of a buzzword and more neutral/descriptive (see [here](https://beauftragte-missbrauch.de/fileadmin/user_upload/Content_Notes/Mit_Content_Notes_auf_sensible_Themen_aufmerksam_machen.pdf) or [here](https://www.cctl.cam.ac.uk/content-notes/what-are-content-notes).)

I guess that's better for our audience. Thanks for reaching

**Review by maehr — APPROVED** [review]

**Pull request merged**